runtime/compiler/ilgen/Walker.cpp

```diff
@@ -48,6 +48,7 @@
 #include "ilgen/J9ByteCodeIlGenerator.hpp"
 #include "infra/Bit.hpp"               //for trailingZeroes
 #include "env/JSR292Methods.h"
+#include "env/j9methodServer.hpp"
```

Comment:
This needs to be protected by `#if defined(J9VM_OPT_JITSERVER)`

Reply:
Added it.